Client/Application/AppDelegate.swift

```diff
@@ -283,3 +288,65 @@ extension AppDelegate {
         menuBuilderHelper?.mainMenu(for: builder)
     }
 }
+
+// MARK: - Application Services History Migration
+// A convinient mapping, `Profile.swift` can't depend
+// on `PlacesMigrationConfiguration` directly since
+// the FML is only usable from `Client` at the moment
+extension PlacesMigrationConfiguration {
+    func into() -> HistoryMigrationConfiguration {
+        switch self {
+        case .disabled:
+            return HistoryMigrationConfiguration.disabled
+        case .dryRun:
+            return HistoryMigrationConfiguration.dryRun
+        case .real:
+            return HistoryMigrationConfiguration.real
+        }
```

Comment:
```suggestion
        case .disabled:
            return .disabled
        case .dryRun:
            return .dryRun
        case .real:
            return .real
        }
```

Reply:
@tarikeshaq curious to know if this will work for a cleanup 🤔 
returning `HistoryMigrationConfiguration` should infer the correct type

Reply:
Good call!! 🚀